torch/nn/modules/rnn.py

```diff
@@ -313,7 +313,7 @@ class RNN(RNNBase):
     function:
 
     .. math::
-        h_t = \tanh(W_{ih} x_t + b_{ih} + W_{hh} h_{(t-1)} + b_{hh})
+        h_t = \tanh(x_t * W_{ih}.T + b_{ih} + h_{(t-1)} * W_{hh}.T + b_{hh})
```

Comment:
nit: to match the [`nn.Linear` docs](https://pytorch.org/docs/stable/generated/torch.nn.Linear.html), please leave out the `*` and use `^T` for the transpose notation.

Reply:
@jbschlosser I have done the required changes.

Reply:
@yashika-git No failures that I've seen! We should wait for all jobs to complete before merging, however.